docs/RELAY.md

```diff
@@ -0,0 +1,41 @@
+# mLRS Documentation: Relay #
+
+([back to main page](../README.md))
+
+**Important: Relay functionality is experimental and currently only available in the branch 'dev-relay', pre-compiled binaries can be found in the firmware folder ([here](https://github.com/olliw42/mLRS/tree/dev-relay/firmware)).**
+
+
+mLRS allows for a relay setup in which a main Tx, Rx pair is used with a secondary Tx, Rx pair to offer flexibility when locating the main Tx module. The relay setup will forward the RC channels from the radio to the flight controller while also sending the link statistics and CRSF sensors of the main Tx, Rx pair back to the radio. A relay setup is beneficial by allowing you to place the main Tx module on your antenna tracker while still being able to control the vehicle from the radio.
+
+For a relay setup is recommended to:
+- Use a lower frequency which is better suited for long range for the main Tx, Rx pair
+- Use different frequencies for the main and secondary Tx, Rx pairs to avoid interference
+- Use a faster mode for the secondary Tx, Rx pair
+
```

Comment:
maybe would should add a note that not all hardware can do that. As a general rule of thumb all F1 secondary Rx can't do it, and exceptions either direction may exist. To avoid that folks get disappointed.

Reply:
Do we want a statement that F1 hardware can't do?  I would think that E28 Dual F1 as Rx could do this (well the inverter hardware is there at least).

Reply:
well, the proper general statement is probably that it is hardware dependend. Currently all F1 don't support it (even if maybe some could)

Reply:
Added a note here.